packages/emotion-utils/src/index.js

```diff
@@ -7,6 +7,13 @@ export function memoize(fn) {
   }
 }
 
+export const KEY_STYLES = '__emotion_styles'
+export const KEY_TARGET = '__emotion_target'
```

Comment:
Could you rename these to `STYLES_KEY` and `TARGET_KEY` respectively so it's easier to immediately understand what they're for?